library/datetime/now_spec.rb

```diff
@@ -5,4 +5,25 @@
   it "creates an instance of DateTime" do
     DateTime.now.should be_an_instance_of(DateTime)
   end
+
+  it "sets the current date" do
+    dt = DateTime.now
+    date = Date.today
+    dt.year.should == date.year
+    dt.mon.should == date.mon
+    dt.day.should == date.day
```

Comment:
This one also needs to use `be_close`, like `(DateTime.now - Date.today).to_f.should be_close(1.0)`.

Reply:
Good point.  Side note: timezones are horrible things. :)